SU-SWS/stanford_profile
# D8CORE-1313: Moving the ol and ul styles to the wysiwyg

# READY FOR REVIEW

# Summary
- Adding Big Paragraph styles to the UL/OL

# Needed By (Date)
- 2/21

# Urgency
- Med

# Steps to Test

1. Pull in the profile changes.
2. Build and pull in the profile.
3. Review an ordered and unordered list in a paragraph to see it has the larger font

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE-1313

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
